rust-runtime/aws-smithy-http-server-python/examples/Makefile

```diff
@@ -1,56 +1,65 @@
-OS := $(shell uname -s)
 SRC_DIR := $(shell git rev-parse --show-toplevel)
 CUR_DIR := $(shell pwd)
 GRADLE := $(SRC_DIR)/gradlew
+WHEELS := $(CUR_DIR)/wheels
 SERVER_SDK_DST := $(CUR_DIR)/pokemon-service-server-sdk
 CLIENT_SDK_DST := $(CUR_DIR)/pokemon-service-client
 SERVER_SDK_SRC := $(SRC_DIR)/codegen-server-test/python/build/smithyprojections/codegen-server-test-python/pokemon-service-server-sdk/rust-server-codegen-python
 CLIENT_SDK_SRC := $(SRC_DIR)/codegen-client-test/build/smithyprojections/codegen-client-test/pokemon-service-client/rust-codegen
 
-SHARED_LIBRARY_DST := $(CUR_DIR)/libpokemon_service_server_sdk.so
-ifeq ($(OS), Darwin)
-	DEBUG_SHARED_LIBRARY_SRC := $(SRC_DIR)/target/debug/libpokemon_service_server_sdk.dylib
-	RELEASE_SHARED_LIBRARY_SRC := $(SRC_DIR)/target/release/libpokemon_service_server_sdk.dylib
-else
-	DEBUG_SHARED_LIBRARY_SRC := $(SRC_DIR)/target/debug/libpokemon_service_server_sdk.so
-	RELEASE_SHARED_LIBRARY_SRC := $(SRC_DIR)/target/release/libpokemon_service_server_sdk.so
-endif
+HAS_MATURIN := $(shell command -v maturin 2> /dev/null)
 
 all: codegen
 
 codegen:
 	$(GRADLE) --project-dir $(SRC_DIR) -P modules='pokemon-service-server-sdk,pokemon-service-client' :codegen-client-test:assemble :codegen-server-test:python:assemble
-	mkdir -p $(SERVER_SDK_DST) $(CLIENT_SDK_DST)
+	mkdir -p $(SERVER_SDK_DST) $(CLIENT_SDK_DST) $(WHEELS)
```

Comment:
Should the wheels dir be created in the `ensure-maturin` task? Or should `build-wheel` and `build-wheel-release` have dependencies on `codegen`? Right now it seems like `make build-wheel` would fail if `make codegen` or `make all` isn't run first.

Reply:
Good catch! Addressed in https://github.com/awslabs/smithy-rs/pull/2025/commits/dd290a852129cf074664a6c969f87e9bb8a768a6